osgar/drivers/rosmsg.py

```diff
@@ -569,17 +569,8 @@ def slot_request_origin(self, timestamp, data):
         cmd = b'request_origin'
         self.bus.publish('cmd', cmd)
 
-    def send_filename(self):
-        try:
-            filename = self.bus.logger.filename  # deep hack
```

Comment:
Oh, this is probably the "Deep Hack" :)

Reply:
BTW it was working for former version

Reply:
I don't understand. What "former version"? Are you saying that it works now? I am not saying that it doesn't work. I am saying it precludes me changing the implementation of bus handle.

Reply:
Sorry, I misunderstood "was not working for zmq-based communication" ... now it is clear, thanks.

Reply:
(the past tense confused me)

Reply:
yeah, sorry, the zmq-based communication is still only in the future :wink: